lab2/token.in

```diff
@@ -1,10 +1,20 @@
 A B C ... Z

```

Comment:
Because each token has to be on separate lines, the letters and digits should respect this as well